opdevnet/wallet/metamask-integration.js

```
@@ -0,0 +1,107 @@
+            signer
+        );
+
+        console.log('Connected to MetaMask and contract instantiated.');
+    } catch (error) {
+        console.error('Error connecting to MetaMask:', error);
+    }
+}
+
+async function depositETH() {
+    let amount = amountInput.value;
+    if (!amount || isNaN(amount) || Number(amount) <= 0) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)